js/views/conversation_view.js

```diff
@@ -241,6 +278,10 @@
             this.model.messageCollection.add(message, {merge: true});
             message.setToExpire();
 
+            if (this.lastSeenIndicator) {
+                this.lastSeenIndicator.increment(1);
+            }
+
```

Comment:
Ah, this is where it gets interesting. We can't rely on the `Conversation` model's `unreadCount` because, as you'll note from the conversation list on the left, it is reset as soon as the conversation is displayed. So the count we need is a unique bit of data stored solely inside of the `LastSeenIndicatorView`. Anyway, I think I'm okay considering the keychange notification as an 'unread message,' since it does occupy space in the conversation.